integrations/jina/src/haystack_integrations/components/reader/jina/__init__.py

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: 2023-present deepset GmbH <info@deepset.ai>
+#
+# SPDX-License-Identifier: Apache-2.0
+from .reader import JinaReaderConnector
+
+__all__ = ["JinaReaderConnector"]
```

Comment:
we should put this component in  
`integrations/jina/src/haystack_integrations/components/converters/`